pom.xml

```diff
@@ -100,7 +100,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-slf4j-impl</artifactId>
+      <artifactId>log4j-slf4j2-impl</artifactId>
```

Comment:
Because of the bug mentioned in the Accumulo's own POM, it may also be necessary to add log4j-core to the runtime dependencies, at least until that bug is fixed.

Reply:
Related - but not part of this PR - the main accumulo project it looks like the modules also need to have log4j-core added to have logging work when running tests.

Reply:
> it may also be necessary to add log4j-core to the runtime dependencies

Are you referring to the main accumulo project or accumulo-examples?

Reply:
> Related - but not part of this PR - the main accumulo project it looks like the modules also need to have log4j-core added to have logging work when running tests.

They only need it if they need to "run" (for tests, yes, for integration tests, yes). Most of the "run" concept for our code is that our application runs out of the distribution tarball, so we don't usually have to care, unless we need it for tests (or the monitor, which I think uses it directly for serializing log events for display on the monitor).

Reply:
> Because of the bug mentioned in the Accumulo's own POM, it may also be necessary to add log4j-core to the runtime dependencies, at least until that bug is fixed.

I may be a bit confused here. The scope of this dependency is `runtime`. Is there more to do here?

Reply:
> > Because of the bug mentioned in the Accumulo's own POM, it may also be necessary to add log4j-core to the runtime dependencies, at least until that bug is fixed.
> 
> I may be a bit confused here. The scope of this dependency is `runtime`. Is there more to do here?

Yes. You need to also add log4j-core as another runtime dependency, because it is not automatically brought in transitively by log4j-slf4j2-impl like it was with log4j-slf4j-impl (it should be, and may yet still be, once upstream fixes the bug they made, but for now, it needs to be listed)

Reply:
> It looks like it already is on main:
> 
> https://github.com/apache/accumulo-examples/blob/516163f3ddadc87f9ffdca61afe08458c61055c2/pom.xml#L96-L100
> 
> Is this what you mean or am I misunderstanding?

Oh, interesting. It looks like @jmark99 added it in ce94cbb3956164742273090d0cec8b5814ff41cc . It probably wasn't needed before. I assumed it was absent, but if it's already there, then there's nothing extra to do here for this.